### PR TITLE
Customtreectrl enhancement to have Window on CustomTreeItem on left of text

### PR DESCRIPTION
**Pull request by HelioGuilherme66**
Adds a the parameter `on_the_right`, defaulted True, to allow positioning the Window in a CustomTreeItem.

[x] - Updated Demo - To see these changes expand item **Hello World**
       - Added a Checkbox to set Window on the left (Unchecked by default)
       - To update the Window on the desired position (Play/Pause image) must enable TREE_ITEM_EXPANDING 
       - To hide these Windows must enable TREE_ITEM_COLLAPSING
       - To update the Window with the animated GIF (Recording image)  on the desired position must enable TREE_ITEM_CHECKED

This is the same as #828 but for wxPy-4.0.x

### CODE REVIEW

**Comment by RobinD42** [comment]
Is this complete? The CustomTreeCtrl sample in the demo doesn't seem to be doing anything different when the option is changed. The Play/Pause images are always displayed on the right of the label. (At least on OSX.)

**Comment by HelioGuilherme66** [comment]
Sorry @RobinD42, no it is not complete. I'll change the title with WIP prefix.

**Comment by HelioGuilherme66** [comment]
@RobinD42 I consider this done now.

File: wx/lib/agw/customtreectrl.py [hunk 1]
```diff
@@ -299,14 +299,14 @@ def __init__(self, parent):
 
 :class:`CustomTreeCtrl` is distributed under the wxPython license.
 
-Latest Revision: Andrea Gavana @ 09 Jan 2014, 23.00 GMT
+Latest Revision: Helio Guilherme @ 02 Jul 2018, 00.10 GMT
 
-Version 2.6
+Version 2.6.1
```
Comment (RobinD42) [inline wx/lib/agw/customtreectrl.py]: This is a significant new feature, so I would call it version 2.7.

**Pull request merged**